### ISSUE TITLE
Output Gaussian components

### ISSUE THREAD

**Issue by MOBO67** [sshaoshuai/MTR#51]
Hello,

in your paper you mention that you use GMM to output the probability p and parameters (µx, µy, σx, σy, ρ) of each Gaussian component for each future time step. How can I access  σx and σy in the evaluation mode? So far I use the following code to get the trajectories with the respective probaibiliy:
torch.no_grad():
    pred_dict = model(batch_dict)
    final_pred_dicts = wd_obj.generate_prediction_dicts(pred_dict)
Thank you in advance for your help.

**Comment by sshaoshuai** [comment]
Please refer to here: 
https://github.com/sshaoshuai/MTR/blob/master/mtr/models/motion_decoder/mtr_decoder.py#L396-L403
and the loss calculation process: 
https://github.com/sshaoshuai/MTR/blob/master/mtr/utils/loss_utils.py#L49-L50. 

So you can access the variances from pred_trajs[..., [2, 3]].

**Comment by hadis-h** [comment]
> 

Is it possible for you to explain how did you load the model ? and print the trajectories ?
